### PR TITLE
More testcases, two bugfixes

### PR DESCRIPTION
**Pull request by DesWurstes**
Hi,

Sorry for not adding WIP. Now it is ready and covers all branches.

EDIT: CI Done

Pushed again with simplified tests and updated bitcore. **ready**

### CODE REVIEW

File: regtest/Dockerfile.bchd [hunk 1]
```diff
@@ -2,8 +2,8 @@
 # and a workspace (GOPATH) configured at /go.
 FROM golang:1.15
 
-RUN apt update \
-    && apt install -y dnsutils
+RUN apt-get update \
+    && apt-get install -y dnsutils
```
Comment (DesWurstes) [inline regtest/Dockerfile.bchd:6]: This is because `apt` is for human consumption while `apt-get` is for scripts.

Comment (jcramer) [reply regtest/Dockerfile.bchd:6]: TIL!

**Comment by DesWurstes** [comment]
@matiu I improved the code to use Schnorr signatures with increased efficiency. A bugfix in bitcore-lib-cash was needed which was merged [today](https://github.com/bitpay/bitcore/pull/3135). Would you rather have this feature in BitcoinFilesJS with the temporary additional patch before a newer bitcore-lib-cash is released or wait up to 1.5 months and enable Schnorr once bitcore-lib-cash has a new version with the fix?

**Comment by jcramer** [comment]
We can just merge whatever you need now, and then fix it later once upstream has it available.

**Pull request merged**